vcf_annotation_tools/vcf_readcount_annotator.py

```diff
@@ -190,44 +197,47 @@ def main(args_input = sys.argv[1:]):
         depth = brct['depth']
         write_depth(entry, sample_name, depth_field, depth)
 
-        #AF - variant allele frequencies
-        if frequency_field not in entry.FORMAT:
-            entry.FORMAT += [frequency_field]
-        vafs = []
-        for alt in alts:
-            alt = alt.serialize()
-            (bam_readcount_position, ref_base, var_base) = parse_to_bam_readcount(start, reference, alt, entry.POS)
-            brct = read_counts.get((chromosome,bam_readcount_position,ref_base), None)
-            if brct is not None:
-                if var_base not in brct:
-                    print("Warning: variant base {} is not present in the bam-readcount entry for variant {} {}. This might indicate that the bam-readcount file doesn't match the VCF.".format(var_base, chromosome, start))
-                    vafs.append(0)
+        if len(brct) != 1: 
```

Comment:
Can you double check that this is actually true for a dictionary with one key-value pair? It might be 2 because a dictionary is just a special array but I’m not 100% and have no way of checking myself right now. You could check `brct.keys()` instead, which I would prefer anyway because it’s clearer. I think you should also add `and brct.keys()[0] == 'depth'` to make this more robust.